src/html.js

```diff
@@ -0,0 +1,43 @@
+// WARN: This is a lightly edited gatsby config file, shouldn't really be edited!
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export default class HTML extends React.Component {
+  render() {
+    return (
+      <html {...this.props.htmlAttributes}>
+        <head>
+          <meta charSet="utf-8" />
+          <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+          <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, viewport-fit=cover, shrink-to-fit=no"
+          />
+          <link
+            href="https://monzo.com/static/css/monzo-framework.min.css"
+            rel="stylesheet"
+          />
+          {this.props.headComponents}
+        </head>
+        <body {...this.props.bodyAttributes}>
+          {this.props.preBodyComponents}
+          <div
+            key={`body`}
+            id="___gatsby"
+            dangerouslySetInnerHTML={{ __html: this.props.body }}
+          />
+          {this.props.postBodyComponents}
+        </body>
+      </html>
+    )
+  }
+}
+
+HTML.propTypes = {
```

Comment:
This is the only place prop-types is used, because this file was generated by Gatsby (I didn't add it!) - hence the note at the top. My aim was to change it as little as possible which is why I didn't touch it - I'll try to change this to `flow` and see what happens 👍